tests/framework/pkg/convert/convert.go

```diff
@@ -0,0 +1,14 @@
+package convert
+
+import (
+	"github.com/rancher/rancher/pkg/api/scheme"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+func ToUnstructured(object runtime.Object) (*unstructured.Unstructured, error) {
```

Comment:
can you use the `MustToUnstructured` from https://github.com/igomez06/rancher/pull/13/files#diff-392d17585207a7ba97127e924c87cac19370e786820d40c5ea503110745a3771R9 to avoid duplication.

Reply:
Yes for sure, once we get your PR merged in I'll update my to use `MustToUnstructured`

Reply:
Fixed